src/docs/patterns/visualizations.mdx

```diff
@@ -123,18 +123,10 @@ There are not as many charting libraries available for React Native applications
 
 There are many libraries and services for data visualizations. Brightlayer UI currently provides themes / utilities for the following libraries:
```

Comment:
you will want to alter the verbiage here too. We don't **provide** themes / utilities anymore but **recommend** them instead